packages/fxa-settings/README.md

```diff
@@ -14,7 +14,55 @@ You can import React components into this project. This is currently restricted
 import HelloWorld from '@fxa-components/HelloWorld';
 ```
 
-**Working with SVGs**
+### Styling components
+
+#### Tailwind
+
+The package uses [Tailwind CSS](https://tailwindcss.com/) for the bulk of its styles. If you're not familiar you're encouraged to [learn more](https://tailwindcss.com/docs/utility-first), but the idea is simple: use predefined classes on elements to layer on individual styles. **You can accomplish most of your styling needs with the classes provided.**
+
+For example, here's what a large, centered, red-text paragraph with a blue background and thick padding would look like:
+
+```html
+<p class="text-lg text-red-600 background-white text-center p-10">
+  Hello, world!
+</p>
+```
+
+It's fairly straightforward once you understand how size, scales and other sequential patterns work. The [Tailwind docs](https://tailwindcss.com/docs) are very helpful in understanding these concepts (and they have an excellent search), and there are plugins for various IDEs ([VS Code](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss), [IntelliJ IDEs](https://plugins.jetbrains.com/plugin/12074-tailwindcss), [Vim](https://github.com/iamcco/coc-tailwindcss)).
+
+Here are a few additional things to keep in mind when developing with Tailwind:
+
+- We overwrite some of the default values provided by Tailwind, such as color and breakpoints. Refer to [tailwind.config.js](./tailwind.config.js) to see how values vary.
+- You do not need to import any custom or built-in styles/dependencies to use Tailwind. Just write your class names.
+- When it comes to time to create a production build we use PostCSS and PurgeCSS to strip out unused styles. PurgeCSS will look through all of the TSX files and identify which class names to keep styles for. In order for this to work properly it's important to avoid dynamic class names:
+
+  ```tsx
+  // Bad
+  const Button = ({ textColor, children }) => {
+    return <button className={`text-${textColor}`}>{children}</button>;
+  };
+  <Button textColor="green-500">Hello, world</Button>;
+
+  // Good!
+  const Button = ({ textColor, children }) => {
+    return <button className={textColor}>{children}</button>;
+  };
+  <Button textColor="text-green-500">Hello, world</Button>;
+  ```
+
+- If you find a class name is getting stripped out erroneously in production builds, it can be [whitelisted](https://purgecss.com/whitelisting.html) in [tailwind.config.js](./tailwind.config.js) or in the stylesheet itself by placing `/* purgecss ignore */` directly above the selector. Please use sparingly.
```

Comment:
This should be pretty rare if we follow the conventions set. I suppose if someone has a weird case and wants to check it builds correctly they can just build the project and view the source to check that class name is present in the output.